### DISCUSSION TITLE
Consider orjson as faster and more robust alternative to ujson

### DISCUSSION THREAD

**Discussion by diegovalenzuelaiturra** [xhluca/bm25s#38] [Ideas & Requests]
Hi,

Should we consider to use `orjson` as an alternative to `ujson` ?

Since `ujson` maintainers encourage their users to migrate to `orjson` because it's both faster and more robust

- https://github.com/ijl/orjson

- https://github.com/ultrajson/ultrajson

Thanks!

**Comment by xhluca** [comment]
That's a good idea! It should be pretty straightforward to add. I will create an issue to track it so we can incorporate in v0.2.

**Comment by xhluca** [comment]
This has been merged to main!
